ush/config_defaults.yaml

```diff
@@ -1177,6 +1183,15 @@ nco:
 # MAKE GRID config parameters
 #-----------------------------
 task_make_grid:
+  execution:
+    nodes: 1
+    cores: 24
+    walltime: 00:20:00
```

Comment:
In general, the new `execution` blocks take their values from what used to be in `parm/wflow/*.yaml` files.